src/logic/operate.js

```diff
@@ -0,0 +1,32 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  let result;
+  const first = Big(numberOne);
+  const second = Big(numberTwo);
+  switch (operation) {
+    case '-':
+      result = first - second;
+      break;
+    case '+':
+      result = first + second;
+      break;
+
+    case '*':
+      result = first * second;
+      break;
+    case '/':
+      if (second === 0) result = undefined;
+      else result = first / second;
+      break;
+    case '%':
+      result = (first / second) * 100;
```

Comment:
You just need to turn a number into a percentage based number here so the ideal thing to do is first / 100 
![see this](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone4/performOperate.png)